src/app/Main/Body/LinkPage/LinkPage-LinkList.component.tsx

```diff
@@ -2,9 +2,9 @@ import React from "react";
 import { useTranslation } from "react-i18next";
 import { List, ListSubheader } from "@material-ui/core";
 import links from "../../../../assets/external_links.json";
-import { LinkListContentComponent as LinkListContent} from "./LinkPage-ListContent.component";
+import { LinkListContentComponent as LinkListContent} from "./LinkList-ListContent.component";
 
-export function LinkListComponent(): JSX.Element[] {
+export function LinkPageListComponent(): JSX.Element[] {

```

Comment:
reject: as i read your pattern this should be: LinkPageLinkListComponent

Reply:
Done.